styles.css

```diff
@@ -1,7 +1,73 @@
-h1 {
-  color: red;
+.flex-container {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
 }
 
-p {
-  color: blue;
+nav {
+  height: 80px;
+  width: 100%;
+  font-size: 30px;
+}
+
+nav label {
+  float: right;
+  line-height: 80px;
+  position: relative;
+  top: 50px;
+  right: 70px;
+}
+
+.logo {
+  position: relative;
+  float: left;
+  top: 70px;
+  left: 20px;
+  font-weight: bold;
+}
```

Comment:
- [ ] Great job so far 👏. Your header looks great but It need more improvement. As the previous reviewer stated, we have to stick to the Figma design given as much as possible. Please compare your header with the Figma design given
**Figma Design**
![image](https://user-images.githubusercontent.com/90222110/176917433-1bad61d3-e7bb-4d5d-b7cb-3a7cc601089a.png)
 **Your Design**                                                       
![image](https://user-images.githubusercontent.com/90222110/176917747-09d1936f-3b18-4386-8d10-201694afa3df.png)
You can observe that there are still a lot to be done. Please correct on your styles. Let me provide you some suggestions below:
- The `nav`  should not have specified fixed height and width because this will make it rigid and distort our design. I would advise you to remove them and use `Flexbox` to align the content inside it. Remember that it is in requirement to use Flexbox for the layout 😃 . Here are the suggested codes. 👇 
```bash
 .nav {
display: flex;
justify-content: space-between;
/**ADDITIONAL STYLES**/
}
```